app/controllers/project.py

```diff
@@ -187,21 +186,20 @@ def delete(self, project_id):
         """
         """
 
-        try:
-            current_user_id = get_jwt_identity()
-            current_user_roles = get_jwt_claims()['roles']
-
-            project = Project.get_by_id(project_id)
+        current_user_id = get_jwt_identity()
+        current_user_roles = get_jwt_claims()['roles']
 
-            if not project:
-                return dict(
-                    status='fail',
-                    message=f'project {project_id} not found'
-                ), 404
+        project = Project.get_by_id(project_id)
```

Comment:
No that doesn't throw an exception, we even use it in other controllers without `try` and `catch`. 
The function `get_by_id()` handles the exception

Reply:
The reason why I pulled it out of the try `block` is because I wanted to call the same variable in the `except` block.
But it is fine in that location